integration

```diff
@@ -340,7 +340,6 @@
   "duhow/hass-aigues-barcelona",
   "duhow/hass-cover-time-based",
   "dummylabs/thewatchman",
-  "DurgNomis-drol/ha_toyota",
```

Comment:
Revert this. This PR should **not** remove any lines.

Reply:
won't we end up with two entries with the same integration name ?

Reply:
That does not matter, there are probably 32 "samsung" integrations also.